src/main/java/teammates/storage/api/InstructorsDb.java

```diff
@@ -365,4 +367,22 @@ protected InstructorAttributes makeAttributes(Instructor entity) {
         return InstructorAttributes.valueOf(entity);
     }
 
+    public List<InstructorAttributes> getAllInstructor() {
```

Comment:
Pack it all to a single method and write comment to say `remove this after data migration is done`

Reply:
Just call ofy() directly from the script; no need to add new code in production just to support the script.